install/installer/pkg/config/v1/experimental/experimental.go

```diff
@@ -58,6 +61,22 @@ type WorkspaceConfig struct {
 			PasswordSecret string   `json:"passwordSecret"`
 		} `json:"redisCache"`
 	} `json:"registryFacade"`
+
+	WorkspaceClasses map[string]WorkspaceClass `json:"classes,omitempty"`
```

Comment:
Shouldn't the `json` field be `workspaceClasses`? as `classes` seems a bit ambiguous? 🤔